dot/core/messages_test.go

```diff
@@ -2,3 +2,265 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 
 package core
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/dot/network"
+	"github.com/ChainSafe/gossamer/dot/peerset"
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/runtime"
+	mocksruntime "github.com/ChainSafe/gossamer/lib/runtime/mocks"
+	"github.com/ChainSafe/gossamer/lib/runtime/storage"
+	"github.com/ChainSafe/gossamer/lib/transaction"
+
+	"github.com/golang/mock/gomock"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	errBestHeader = errors.New("best header error")
+	errGetRuntime = errors.New("get runtime error")
+	errTrieState  = errors.New("trie state error")
```

Comment:
Why not just have a `errTest = errors.New("dummy error")` if it's only used for tests? Otherwise use errors defined in the production code.

Reply:
The only reason I could think is if we wanted to make sure we are asserting specific errors, but I think this should be fine to do